frontend/src/App.vue

```diff
@@ -242,19 +242,18 @@ export default class App extends Vue {
 
     // Lifecycle hook
     created() {
-        if (!this.$store.getters["security/isNoSecurityActive"]) {
-            SsoUserInfoService.getUserInfo()
-                .then((ssoUserInfoResponse: SsoUserInfoResponse) => {
-                    this.$store.dispatch(
-                        "user/setSsoUserInfoResponse",
-                        ssoUserInfoResponse
-                    );
-                    this.loggedInUser = this.$store.getters["user/getName"];
-                })
-                .catch(() => {
-                    return false;
-                });
-        }
+        SsoUserInfoService.getUserInfo()
```

Comment:
Bitte wieder zurückbauen. Wenn es jetzt zu einem Fehler beim Laden des UserInfo kommt, wird automatisch immer der NoSecModus aktiviert und jeder kann die Anwendung verwende. Das ist nicht gut in der Produktion